nmosconnection/service.py

```diff
@@ -21,7 +21,6 @@
 
 import signal
 import gevent
-import sys
 
 from nmoscommon.httpserver import HttpServer
 from api import ConnectionManagementAPI, QUERY_APINAME, QUERY_APIVERSION, DEVICE_ROOT
```

Comment:
Not a comment on this PR as such, but I suspect these constants were copied from the Query API originally, so could ideally be renamed for the CM API

Reply:
Yup, almost certainly what happened. I'll fix it next time I do a change on the repo